_posts/2017-02-02-Rust-1.15.md

```diff
@@ -0,0 +1,273 @@
+---
+layout: post
+title: "Announcing Rust 1.15"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.15.0. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+As always, you can [install Rust 1.15.0][install] from the appropriate page on our
+website, and check out the [detailed release notes for 1.15.0][notes] on GitHub.
+1443 patches were landed in this release.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1150-2017-02-02
+
+### What's in 1.15.0 stable
+
+Rust 1.15 sees an _extremely_ long-awaited feature land on stable: custom
+derive! To review, in Rust, you could automatically implement a trait through
```

Comment:
Hm, next para also uses the past tense in this way. It feels awkward to me, but not necessarily wrong.